docs/building-a-form/adding-widgets-and-definitions.md

```diff
@@ -13,19 +13,21 @@ These common widgets are included in the us-forms-system by default. Set these w
 
 To use a widget, set `ui:widget` for a field to the name of the widget. Widgets are located in [/src/js/widgets](../src/js/widgets).
 
-Widget       | Default schema type or format
+Widget       | How it's used
 ------------ | -------------
-`ArrayCountWidget` |
-`CheckboxWidget` | type: `boolean`
-`CurrencyWidget` |
-`DateWidget` |
-`EmailWidget` |
-`PhoneNumberWidget`
-`RadioWidget` |
-`SelectWidget` | type: `string` with an enum property
-`SSNWidget` |
-`TextWidget` | type: `string`
-`YesNoWidget` |
+`ArrayCountWidget` | In the 'uiSchema', specify `'ui:widget': ArrayCountWidget` for the given field
```

Comment:
Some of these descriptions are using straight quotes and some backticks. I think they all need to be backticks.